app/assets/stylesheets/components/_index.scss

```diff
@@ -7,3 +7,4 @@
 @import "footer";
 @import "navbar";
 @import "short-card-grid";
+@import "card-listing-l";
```

Comment:
This code looks great. I read that adding each @import in alphabetical order makes it easier to find components while working on teams. Should we use this method?

Reply:
yes sounds great